src/main/webapp/app/guided-tour/guided-tour.module.ts

```diff
@@ -0,0 +1,32 @@
+import { CUSTOM_ELEMENTS_SCHEMA, NgModule, ErrorHandler } from '@angular/core';
+import { ModuleWithProviders } from '@angular/compiler/src/core';
+import { JhiLanguageService } from 'ng-jhipster';
+
+import { JhiLanguageHelper } from 'app/core';
+import { ArtemisSharedModule } from 'app/shared';
+import { GuidedTourService } from './guided-tour.service';
+import { GuidedTourComponent } from './guided-tour.component';
+
+@NgModule({
+    declarations: [GuidedTourComponent],
+    imports: [ArtemisSharedModule],
+    exports: [GuidedTourComponent],
+    entryComponents: [GuidedTourComponent],
+    schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
please remove the CUSTOM_ELEMENTS_SCHEMA.
Have you merged develop? we recently remove all entries of CUSTOM_ELEMENTS_SCHEMA